Platform/TigerlakeBoardPkg/AcpiTables/SsdtRtd3/Rtd3PcieWwan.asl

```diff
@@ -0,0 +1,290 @@
+/** @file
+  ACPI RTD3 SSDT table for PCIe WWAN

```

Comment:
I think it is not necessary to add this file for S0ix support.

Reply:
Done removed it, thanks.